E-Commerce/Controllers/AccountController.cs

```diff
@@ -0,0 +1,86 @@
+using E_Commerce.Models.Identity;
+using E_Commerce.Services.Identity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace E_Commerce.Controllers
+{
+    public class AccountController : Controller
+    {
+        private readonly IUserService userService;
+
+        public AccountController ( IUserService userService )
+        {
+            this.userService = userService;
+        }
+
+        [Authorize(Roles = "Site Owner")]
```

Comment:
You'll access denied for the My Account page unless you have assigned the role "Site Owner to your user. Ideally I think you'd want any user to be able to see their own Account profile page.